Added the network option for voluntary-exit command (#487)
Consensys/doc.teku: docs/reference/cli/subcommands/voluntary-exit.md

Include validator keys managed using the [key manager APIs](../../../how-to/use-external-signer/manage-keys.md). The default is `false`.

## `network`

<!--tabs-->

# Syntax

```bash
teku voluntary-exit --network=<NETWORK>
```

# Example

```bash
teku voluntary-exit --network=mainnet
```

# Environment variable

```bash
TEKU_NETWORK=mainnet
```

# Configuration file

```bash
network: "mainnet"
```

<!--/tabs-->

> Predefined network configuration. The default is unset, as the network specification will be read from the beacon-api unless specified.

Reviewer: ```suggestion
Predefined network configuration. There is no default value, because Teku reads the network specification from the Beacon API unless specified.
```
